azbfs/zc_mmf_unix.go

```diff
@@ -7,8 +8,10 @@ import (
 	"syscall"
 )
 
+//nolint:unused
 type mmf []byte
 
+//nolint:unused,deadcode
```

Comment:
this I left since it seems to be necessary to have a linux/darwin impl. But is it actually unused? if so I can delete

Reply:
Deleted both zc_mmf files in azbfs